examples/generic/can/main.cpp

```diff
@@ -26,82 +35,67 @@ displayMessage(const modm::can::Message& message)
 	static uint32_t receiveCounter = 0;
 	receiveCounter++;
 
-	MODM_LOG_INFO<< "id  =" << message.getIdentifier();
+	MODM_LOG_INFO << "id  =" << message.getIdentifier();
 	if (message.isExtended()) {
-		MODM_LOG_INFO<< " extended";
+		MODM_LOG_INFO << " extended";
 	}
 	else {
-		MODM_LOG_INFO<< " standard";
+		MODM_LOG_INFO << " standard";
 	}
 	if (message.isRemoteTransmitRequest()) {
-		MODM_LOG_INFO<< ", rtr";
+		MODM_LOG_INFO << ", rtr";
 	}
-	MODM_LOG_INFO<< modm::endl;
+	MODM_LOG_INFO << modm::endl;
 
-	MODM_LOG_INFO<< "dlc =" << message.getLength() << modm::endl;
+	MODM_LOG_INFO << "dlc =" << message.getLength() << modm::endl;
 	if (!message.isRemoteTransmitRequest())
 	{
 		MODM_LOG_INFO << "data=";
 		for (uint32_t i = 0; i < message.getLength(); ++i) {
-			MODM_LOG_INFO<< modm::hex << message.data[i] << modm::ascii << ' ';
+			MODM_LOG_INFO << modm::hex << message.data[i] << modm::ascii << ' ';
 		}
-		MODM_LOG_INFO<< modm::endl;
+		MODM_LOG_INFO << modm::endl;
 	}
-	MODM_LOG_INFO<< "# received=" << receiveCounter << modm::endl;
+	MODM_LOG_INFO << "# received=" << receiveCounter << modm::endl;
 }
 
 // ----------------------------------------------------------------------------
 int
 main()
 {
 	Board::initialize();
-
-	// Initialize Usart
-	Usart2::connect<GpioOutputA2::Tx>();
-	Usart2::initialize<Board::systemClock, 115200>();
-
+	Board::Leds::setOutput();
+#ifndef MODM_BOARD_HAS_LOGGER
+    Generic::Uart::connect();
+#endif
 	MODM_LOG_INFO << "CAN Test Program" << modm::endl;
 
-	MODM_LOG_INFO << "Dividing filter bank..." << modm::endl;
-	CanFilter::setStartFilterBankForCan2(14);
-
 	MODM_LOG_INFO << "Initializing Can1..." << modm::endl;
```

Comment:
`Generic::Can` is not necessarily `Can1`.

Reply:
I know, I'm not finished refactoring. The CAN example in general is pretty messy, but I'd like to add Can2 as well, when it's available.